go.mod

```diff
@@ -4,7 +4,13 @@ go 1.22.2
 
 require github.com/spf13/cobra v1.8.0
 
+require golang.org/x/sys v0.0.0-20220715151400-c0bba94af5f8 // indirect
+
 require (
+	github.com/Masterminds/semver/v3 v3.2.1
 	github.com/inconshreveable/mousetrap v1.1.0 // indirect
+	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/pflag v1.0.5 // indirect
+	go.k6.io/xk6 v0.11.0
```

Comment:
I think k6build should not depend on xk6, rather the necessary logic should be moved from xk6 to k6build....

Reply:
I will do this in a follow up PR